OfficeMicroService/Application/Services/IOfficeServices.cs

```diff
@@ -0,0 +1,18 @@
+﻿using OfficeMicroService.Data.Enum;
+using OfficeMicroService.Data.Models.DTO;
+
+namespace OfficeMicroService.Application.Services
+{
+    public interface IOfficeServices
+    {
+        Task<List<OfficeIdDTO>> GetAsync();
+        Task<OfficeIdDTO> GetAsync(string id);
```

Comment:
rename this method or the method above to better suit the context (the problem that naming is the same for one entity and list)